packages/client/src/helpers/client/formatError.ts

```diff
@@ -73,14 +74,18 @@ export const adaptError = (error: {
     return error;
   }
 
+  const extraParameters: Record<string, unknown> = {};
```

Comment:
```suggestion
  const extraParameters: {
    transportLayerErrorCode?: unknown, // I really don't know the correct type here
    [k: string]: unknown,
  } = {};
```

Just because we know about the `transportLayerErrorCode`, makes sense? 😇